stream_alert_cli/outputs.py

```diff
@@ -97,23 +98,24 @@ def encrypt_and_push_creds_to_s3(region, bucket, key, props):
         return True
 
     creds_json = json.dumps(creds)
-    enc_creds = kms_encrypt(region, creds_json)
+    enc_creds = kms_encrypt(region, creds_json, kms_key_alias)
     return send_creds_to_s3(region, bucket, key, enc_creds)
 
 
-def kms_encrypt(region, data):
+def kms_encrypt(region, data, kms_key_alias):
     """Encrypt data with AWS KMS.
 
     Args:
         region [string]: AWS region to use for boto3 client
         data [string]: json string to be encrypted
+        kms_key_alias [string]: The KMS key alias to use for encryption of S3 objects
 
     Returns:
         [string] Encrypted ciphertext data blob
     """
     try:
         client = boto3.client('kms', region_name=region)
-        response = client.encrypt(KeyId='alias/stream_alert_secrets',
+        response = client.encrypt(KeyId=kms_key_alias,
```

Comment:
++ for this!!
Question though - should we keep the `alias/` prefix stuff here to avoid having to have it in a bunch of different places?